test/integration-new/cni/eni_config/config_suite_test.go

```diff
@@ -0,0 +1,77 @@
+package env_vars
```

Comment:
Directory structure `cni/eni_config` can be misleading as this is testing bunch of environment variable and not the ENIConfig Custom Resource. Also consider placing it another directory alongside cni if you think it doesn't fit in the cni directory.

Reply:
Now this has env_vars and IP leak test. So I was thinking what would be the suitable name

Reply:
But this also has IPAMD logging test and in future may have some other tests. I think it should have some generic name.

Reply:
I will change it, I am thinking what should be a good name :)